Sources/Public/CombineExtension.swift

```diff
@@ -63,8 +63,26 @@
                     Pretty.prettyPrint($0, option: option, to: &s)
                     _print(s, type: .output, terminator: "")
                 }
-            }, receiveCompletion: {
-                _print("receive \($0)", type: .completion)
+            }, receiveCompletion: { completion in
+                switch completion {
+                case .finished:
+                    _print("receive finished", type: .completion)
+                case let .failure(error):
+                    switch format {
```

Comment:
This code pattern is same of L53-65.

I would like to extract local method as follows:

```swift
_prettyPrint(label: "receive failure", type: .completion) // `format` is captured by local method.
```

What do you think?

Reply:
extracted it